Dockerfile

```diff
@@ -17,25 +20,25 @@ ARG IMAGE_VERSION=""
 
 # Custom build options:
 #  Disable certain image layers using build env variables if desired
-ARG IMAGE_LAYER_SYS_EXT
-ARG IMAGE_LAYER_PERL_EXT
-ARG IMAGE_LAYER_DEV
-ARG IMAGE_LAYER_PERL_CPAN
-ARG IMAGE_LAYER_PERL_CPAN_EXT
-ARG IMAGE_LAYER_PYTHON
-ARG IMAGE_LAYER_PYTHON_EXT
-ARG IMAGE_LAYER_NODEJS
-ARG IMAGE_LAYER_NODEJS_EXT
+ARG IMAGE_LAYER_SYS_EXT="1"
+ARG IMAGE_LAYER_PERL_EXT="1"
+ARG IMAGE_LAYER_DEV="1"
+ARG IMAGE_LAYER_PERL_CPAN="1"
+ARG IMAGE_LAYER_PERL_CPAN_EXT="1"
+ARG IMAGE_LAYER_PYTHON="1"
+ARG IMAGE_LAYER_PYTHON_EXT="1"
+ARG IMAGE_LAYER_NODEJS="1"
+ARG IMAGE_LAYER_NODEJS_EXT="1"
 
 # Custom installation packages
-ARG APT_PKGS
-ARG CPAN_PKGS
-ARG PIP_PKGS
-ARG NPM_PKGS
+ARG APT_PKGS=""
+ARG CPAN_PKGS=""
+ARG PIP_PKGS=""
+ARG NPM_PKGS=""
```

Comment:
be careful changing this as on travis, it wouldn't work the way it is now. May be different on Github Actions, to be verified